README.md

```diff
@@ -67,11 +67,11 @@ If you run into an error that says `pragma and pragmaFrag cannot be set when run
 
 ### Components and Controls
 
-Our component documentation is hosted on the [FluentUI documentation](https://developer.microsoft.com/fluentui).
+Our component documentation can be found in SPEC.md files for each component. The current list can be found in our [Wiki's sidebar](https://github.com/microsoft/fluentui-react-native/wiki). They will be uploaded to a website at a future time.
```

Comment:
Seems like this list only includes SPEC.md files in components that are in the components folder and not in experimental, just wanted to check that that was intentional? I wrote up a SPEC.md file for Shadow and wondering if that should be showing up somewhere

Reply:
The sidebar has been updated with spec files of components we've released, which are those that we've moved out of the experimental folder. That's intentional because you could be in the middle of building a component which already has a SPEC file, but the SPEC file may be incomplete. A component is only considered finished once released and migrated.

I believe Shadow was held off for some reason, something around wanting to wait until it was ready for all platforms?